Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/AnimGraph/BlendGraphWidget.cpp

```diff
@@ -155,6 +168,30 @@ namespace EMStudio
             return;
         }
 
+        // calculate the drop position
+        QPoint localPos = LocalToGlobal(event->pos());
+
+        // Dragged from node palette
+        if (event->mimeData()->hasFormat(BlendGraphMimeEvent::BlendGraphMimeEventType))
+        {
+            GraphCanvas::GraphCanvasMimeContainer mimeContainer;
+            const QByteArray ba = event->mimeData()->data(BlendGraphMimeEvent::BlendGraphMimeEventType);
```

Comment:
uber nit: We try to avoid single or double letter variable names so a name more inline with what the variable is rather than the type itself would be great, so perhap `blendGraphMimeData` or something similar?

Reply:
Done